mjsdevs/vitae-web
# Feat/login form

## What I did:
- Added email input
- Added user icon on email input
- Added password input
- Added key icon on the password input
- Added sign-in button
- Added "forgot password?" option
- Added sign up option
- Ensured that the form shows an error message for the user if the fields are not populated correctly

## How to test:
- Run `yarn` to install the dependencies
- Run `yarn start` to start the project
- Then go to http://localhost:3000, there you should see the Login page
- Try Signing-in with an invalid e-mail or a password smaller than 3 digits.
- You should see a friendly error message

resolves #1
